common/shareddb.cpp

```diff
@@ -964,7 +964,7 @@ void SharedDatabase::LoadItems(void *data, uint32 size, int32 items, uint32 max_
 
 		// Flags
 		item.ArtifactFlag = Strings::ToBool(row[ItemField::artifactflag]);
-		item.Attuneable = !disable_attuneable && Strings::ToInt(row[ItemField::attuneable]) != 0;
+		item.Attuneable = !disable_attuneable && Strings::ToBool(row[ItemField::attuneable]) != 0;
```

Comment:
I wonder if these can have wonky behavior since we’re comparing a Boolean to an integer. Should we just check if it’s true instead here and elsewhere I see it used here?

Reply:
The != 0 should just be removed if you’re going to use tobool